requirements.txt

```diff
@@ -5,3 +5,4 @@ flake8-variables-names==0.0.5
 pep8-naming==0.13.2
 django-debug-toolbar==3.2.4
 django-crispy-forms==1.14.0
+crispy_bootstrap4
```

Comment:
and why is it needed? what does that change? what's the point

Reply:
To pin your app to lib version. It is good practice, otherwise your app will retrieve latest available, and in case of changes in lib, your app could go down